README.md

```diff
@@ -45,8 +45,4 @@ Before publishing the typescript modules, do a manual compilation step via `npm
 
  `npm run bootstrap` - perform boostrap for all modules
 
- `npm run start` - run top level applications
-
- `npm run build` - execute the build command for all modules, compiling TypeScript sources to JavaScript
-
- `npm run cleanInstall` - perform install without executing additional scripts
```

Comment:
Test is above. Clean install is no longer required as we do not compile typescript in this repository.